<?xml version="1.0"?>
<launch>
    <arg name="tag" default="38"/>
    <arg name="task_name" default="taskA.yaml"/>

    <!-- set up task assignment and evaluation -->
    <node name="" pkg="RGMC_In-Hand_Manipulation_2024" type="task_eval.py" output="screen">
        <param name="task" value="1"/>
        <param name="tag" value="$(arg tag)"/>
        <param name="task_file" value="$(find RGMC_In-Hand_Manipulation_2024)/task/$(arg task_name)"/>
        <param name="cube_file" value=""/>
    </node>

</launch>